components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/EmailTemplateManagerImpl.java

```diff
@@ -477,33 +510,107 @@ private String resolveNotificationChannel(String notificationChannel) {
         }
     }
 
+    /**
+     * Create a registry resource instance of the notification template.
+     *
+     * @param notificationTemplate Notification template
```

Comment:
Put full stop end of comment

Reply:
`.` has been added to the method level comment. We do not add `.` at the end of param descriptions.